dashboard-frontend/src/types/study.ts

```diff
@@ -0,0 +1,33 @@
+export interface Study {
+  id: string;
+  name: string;
+  slug: string;
+  createdAt: string;
+  progress: StudyProgress;
+  strata: Strata[];
+}
+
+interface StudyProgress {
+  currentParticipants: number;
+  expectedParticipants: number;
+  currentAverageDeviation: number;
+  expectedAverageDeviation: number;
+  date: string;
+  averageDeviationOverTime: DataPoint[];
+}
+
+interface Strata {
+  id: string;
+  percentageDesired: number;
+  currentPercentage: number;
+  remainingPercentage: number;
```

Comment:
In `StudyProgress` we're calling this "devation" and here we're calling it "remaining" -- we should pick one word. 

Probably, "deviation" is better because it can be above or below!

Reply:
I renamed it to **percentageDeviation**

Reply:
I agree, it's clear!